server/routerlicious/packages/services-utils/src/auth.ts

```diff
@@ -22,11 +22,19 @@ import type { Provider } from "nconf";
 export function validateTokenClaims(
     token: string,
     documentId: string,
-    tenantId: string): ITokenClaims {
+    tenantId: string,
+    ignoreDocumentId = false): ITokenClaims {
     const claims = jwt.decode(token) as ITokenClaims;
+    if (!claims) {
+        throw new NetworkError(403, "Missing token claims.");
+    }
 
-    if (!claims || claims.documentId !== documentId || claims.tenantId !== tenantId) {
-        throw new NetworkError(403, "DocumentId and/or TenantId in token claims do not match request.");
+    if (claims.tenantId !== tenantId) {
+        throw new NetworkError(403, "TenantId in token claims does not match request.");
+    }
+
+    if (!ignoreDocumentId && claims.documentId !== documentId) {
```

Comment:
This is a very minor comment, but should we instead use `requireDocumentId` and set it default to `true` instead of `ignoreDocumentId` and use the default `false`? The reason I ask is because every time we use `ignoreDocumentId` we have to add a `!` in front of it.